luarules/gadgets/scavengers/Blueprints/BYAR/blueprint_tiers.lua

```diff
@@ -1,20 +1,12 @@
 local UDN = UnitDefNames
 local wallChance = 0
-local ScavengerTeamID = Spring.GetGaiaTeamID()
-local teams = Spring.GetTeamList()
-for i = 1, #teams do
-	local luaAI = Spring.GetTeamLuaAI(teams[i])
-	if luaAI and luaAI ~= "" and string.sub(luaAI, 1, 12) == 'ScavengersAI' then
-		ScavengerTeamID = i - 1
-		break
-	end
-end
+local gaiaTeamID = Spring.GetGaiaTeamID()
```

Comment:
```suggestion
local gaiaTeamID = Spring.Utilities.GetScavTeamID() or Spring.GetGaiaTeamID()
```
The original code is using a different team ID if Scavenger are present.